### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by kachakot**

### CODE REVIEW

File: .storybook/config.js [hunk 1]
```diff
@@ -8,10 +8,10 @@ import GlobalStyles from '../src/common/GlobalStyles'
 
 addDecorator((story) => (
   <ThemeProvider theme={theme}>
-    <Fragment>
+    <>
       <GlobalStyles/>
```
Comment (rgolubowicz) [inline .storybook/config.js]: add space before closing tag
`<GlobalStyles />`

File: src/common/DynamicComponent.js [hunk 2]
```diff
@@ -21,15 +22,22 @@ const StyledDynamicComponent = styled(tag)`
 `
 
 const DynamicComponent = ({
-  tag,
+  newTag,
   children,
   ...props }) => {
```
Comment (rgolubowicz) [inline src/common/DynamicComponent.js]: add new line after `...props`

File: src/components/BulletList/BulletList.js [hunk 3]
```diff
@@ -2,13 +2,13 @@ import React from 'react'
...
 import {
   BulletListStyled,
-  BulletListItemStyled
+  BulletListItemStyled,
 } from './styledComponents'
 import { Typo6 } from '../Typography/Typography'
 
-const BulletList = (props) => {
+const BulletList = props => {
   const {
```
Comment (rgolubowicz) [inline src/components/BulletList/BulletList.js]: do we need multi lines here if it's really short?

File: src/components/ButtonLink/styledComponents.js [hunk 4]
```diff
@@ -1,7 +1,17 @@
 import styled, { css } from 'styled-components'
```
Comment (rgolubowicz) [inline src/components/ButtonLink/styledComponents.js]: remove this empty line

File: src/components/Color/Color.js [hunk 5]
```diff
@@ -10,54 +9,51 @@ const Color = () => {
       flexWrap: 'wrap',
```
Comment (rgolubowicz) [inline src/components/Color/Color.js]: get rid of indents in that file

File: src/components/Wrapper/Wrapper.js [hunk 6]
```diff
@@ -1,25 +1,29 @@
...
-  return (
-    <WrapperStyled style={{
-      ...otherProps
-    }}>
-      {children}
-    </WrapperStyled>
-  )
-}
+}) => (
+  <WrapperStyled style={{
```
Comment (rgolubowicz) [inline src/components/Wrapper/Wrapper.js]: do we need destructuring here? Also rename otherProps to styles

File: src/common/DynamicComponent.js (hunk 2 above)
Comment (kachakot) [reply src/common/DynamicComponent.js]: done

File: src/components/BulletList/BulletList.js (hunk 3 above)
Comment (kachakot) [reply src/components/BulletList/BulletList.js]: done

File: src/components/ButtonLink/styledComponents.js (hunk 4 above)
Comment (kachakot) [reply src/components/ButtonLink/styledComponents.js]: done

File: src/components/Color/Color.js (hunk 5 above)
Comment (kachakot) [reply src/components/Color/Color.js]: hmmm I am not sure what should I do here

File: src/components/Wrapper/Wrapper.js (hunk 6 above)
Comment (kachakot) [reply src/components/Wrapper/Wrapper.js]: done

File: .storybook/config.js (hunk 1 above)
Comment (kachakot) [reply .storybook/config.js]: done

File: src/components/Color/Color.js (hunk 5 above)
Comment (kachakot) [reply src/components/Color/Color.js]: it should be done

File: src/components/Field/Radio/styledComponents.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
-import styled from "styled-components"
+import styled from 'styled-components'
```
Comment (rgolubowicz) [inline src/components/Field/Radio/styledComponents.js]: do we need this empty line? remove it :-)

File: src/components/Link/Link.js [hunk 8]
```diff
@@ -18,27 +18,36 @@ const Link = ({
   <LinkStyled
     {...rest}
   >
-    {withGithub &&
-      <LinkIconStyled>
-        <GitHub fontSize="16px"/>
-      </LinkIconStyled>
-    }
+    {withGithub
```
Comment (rgolubowicz) [inline src/components/Link/Link.js]: it can looks like
```
{withGithub && (
  ...
  ...
)}
```

File: src/components/Project/Project.js [hunk 9]
```diff
@@ -36,33 +36,39 @@ const Project = ({
...
-            label="Read more"
-          /> :
-          <Link
-            withArrow
-            iconColor={theme.color.red}
-            href="#"
-            label="Read more"
-          />
-        }
+        {reversed
```
Comment (rgolubowicz) [inline src/components/Project/Project.js]:
```
{reversed && (
  ...
  ...
)}
```

File: src/components/Project/styledComponents.js [hunk 10]
```diff
@@ -47,13 +47,15 @@ const ProjectActionStyled = styled.div``
 const ProjectStyled = styled.div`
   position: relative;
   width: 100%;
-  padding: ${props => props.theme.gutter.gutter32} ${props => props.theme.gutter.gutter48};
+  padding: (${props => props.theme.gutter.gutter32}
```
Comment (rgolubowicz) [inline src/components/Project/styledComponents.js]:
```
padding: (
  ${props => props.theme.gutter.gutter32}
  ...
)

```

File: src/components/Link/Link.js (hunk 8 above)
Comment (kachakot) [reply src/components/Link/Link.js]: done

File: src/components/Field/Radio/styledComponents.js (hunk 7 above)
Comment (kachakot) [reply src/components/Field/Radio/styledComponents.js]: done

File: src/components/Project/styledComponents.js (hunk 10 above)
Comment (kachakot) [reply src/components/Project/styledComponents.js]: done

**Pull request merged**